design-cfps/README.md

```diff
@@ -0,0 +1,51 @@
+This repo contains CFP design proposals for features impacting repos across the
+Cilium Github organization.
+
+# Purpose of CFPs
+
+The purpose of a CFP is to allow community members to gain feedback
+on their designs from the Committers before the community member commits to
+executing on the design. By going through the design process, developers gain a
+have a high level of confidence that their designs are viable and will be
+accepted.
+
+NOTE: This is process is not mandatory. Anyone can execute on their own design
+without going through this process and submit code to the respective repos.
+However, depending on the complexity of the design and how experienced the
+developer is within the community, they could greatly benefit from going through
+this design process first. The risk of not getting a design proposal approved
+is that a developer may not arrive at a viable architecture that the community will
+accept.
+
+# How to create CFPs
+
+To create a CFP, it is recommended to use the `CFP-YYYY-MM-DD-template.md`
+file as an outline. The structure of this template is meant to provide a starting
+point for people. Feel free to edit and modify your outline to best fit your
+needs when creating a proposal.
```

Comment:
Maybe something on the naming format we expect? Are we going to go with "have an issue first, and name this after the issue number" like KEPs etc, or should we do `CFP-YYYY-MM-DD-subject.md`, where the date is the day the discussion is opened? I lean towards the former (it makes things more trackable), but not strongly.

Reply:
All of the CFPs I've seen so far use the latter format so I'll stick with that for now unless we decide something else.